mono/mini/exceptions-amd64.c

```diff
@@ -887,7 +887,7 @@ mono_arch_handle_altstack_exception (void *sigctx, MONO_SIG_HANDLER_INFO_TYPE *s
 	MonoException *exc = NULL;
 	gpointer *sp;
 	int frame_size;
-	MonoContext *copied_ctx;
+	ucontext_t *copied_ctx;
```

Comment:
unfortunately, `ucontext_t` is not portable 😕 That's the whole reason why we do the whole exercise with `MonoContext`.

Reply:
I followed what was already done in the -ppc version of the code under the assumption that it's ok to do if it's limited to the platform specific file.

It was the least intrusive change. I do agree that it's probably better to propagate MonoContext, but I wanted to keep the change small.